Makefile

```diff
@@ -166,6 +166,7 @@ MK_INCL_DIR_INST          := $(INSTALL_INCDIR)/blis
 # Set the path to the subdirectory of the share installation directory.
 MK_SHARE_DIR_INST         := $(INSTALL_SHAREDIR)/blis
 
+PC_SHARE_DIR_INST         := $(INSTALL_SHAREDIR)/pkgconfig
```

Comment:
Isn't this supposed to be `$(INSTALL_LIBDIR)/pkgconfig`?

Reply:
It could be, but since the rest of the `blis` config files are installed in `share`, I used the same to be consistent.

Reply:
pkg-config will not find `blis` if it's not installed into `$(INSTALL_LIBDIR)/pkgconfig` right? (Unless the user manually sets `PKG_CONFIG_PATH`)

Reply:
Right. Thanks for bringing that up. I was only testing it from within a `CMake` project and that works either way with `CMake`. How about adding a `libdir=@prefix@/share` to `blis.pc.in`. It should automatically pick up from `datadir/pkgconfig`  as well right ? Alternatively, I could also change it to `$(INSTALL_LIBDIR)/pkgconfig`, but I would prefer @devinamatthews comment on this one before I do that.

Reply:
See comment [here](https://github.com/flame/blis/pull/473#issuecomment-758802730).